examples/connector/python/connector/main.py

```diff
@@ -0,0 +1,176 @@
+import grpc
+from concurrent import futures
+import json
+import datetime
+
+
+import connector_sdk_pb2_grpc as connector__sdk__pb2__grpc
+import common_pb2 as common__pb2
+import connector_sdk_pb2 as connector__sdk__pb2
+
+
+class ConnectorService(connector__sdk__pb2__grpc.ConnectorServicer):
+    def ConfigurationForm(self, request, context):
+        form_fields = common__pb2.ConfigurationFormResponse(schema_selection_supported=True,
+                                                           table_selection_supported=True)
+        api_key_field = form_fields.fields.add()
```

Comment:
we can add and initialise the fields in a single line, like
`form_fields.fields.add(name="apiKey", label="API Key", required=True, text_field=common__pb2.TextField.PlainText)`, since the method support kwargs